### DISCUSSION TITLE
Access to CAD files?

### DISCUSSION THREAD

**Discussion by MikeMcPherson** [alanbjohnston/CubeSatSim#155] [Ideas]
Hi everyone

What are the chances we could get the original CAD files for the frame posted to Github, please?  I would like to make some minor modifications (change to heat-set nuts, for example).  Importing the STL file and "gluing and carving" is certainly possible, but it would be easier with the originals.

Thanks.  Mike KQ9P

**Comment by alanbjohnston** [comment]
Hi Mike, we didn’t create the frame, we used an exiting one on Thingiverse. We emailed the creator asking if he would share the CAD files but no reply so far. Some others are modifying the frame and have said they will share the CAD files. I’ll post a link as soon as I get it.

**Reply by MikeMcPherson** [reply]
No problem, it's not a big deal.  Thanks!  Mike KQ9P
